lib/pages/race/moderator/race_moderator_page.dart

```diff
@@ -73,11 +74,11 @@ class _RaceModeratorPageState extends ConsumerState<RaceModeratorPage> {
   @override
   Widget build(BuildContext context) {
     final teamScores = ref.watch(teamScoresProvider(eventId));
+    Map<String, LatLng> teamPositions =
+        ref.watch(teamPositionNotifierProvider(eventId));
     // todo do something with below
     // ignore: unused_local_variable
```

Comment:
are these comments still accurate?

Reply:
yes, the variable in the next line is not used haha